federation/deploy/config.json.sample

```diff
@@ -16,7 +16,7 @@
     },
     "phase2": {
       "docker_registry": "gcr.io/google-containers",
-      "kubernetes_version": "v1.3.0"
+      "kubernetes_version": "v1.4.0"
```

Comment:
> Is this a required param?

It is a required parameter, yes.

> Will it use latest if i just skip it?

No. We want the users to explicitly state what they want. And this is a sample file, so it should show all the fields for demonstration.

> Or can I set "latest" here?

You can. But we don't want to encourage that pattern. This will be used in a Deployment. And the Deployment controller can't pick up new versions if you set it to latest.